scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -421,6 +428,50 @@ class ExperimentalScanner(
             ScanResult(provenance, scanner.details, summary)
         }
     }
+
+    private fun createFileArchives(nestedProvenances: Map<Package, NestedProvenance>) {
+        // TODO: The archives are currently created in a way compatible with the existing implementation in the
+        //       LocalScanner. This allows to keep using existing file archives without changing the logic used to
+        //       access those archives in the reporter. To achieve this nested provenances are downloaded recursively.
```

Comment:
Is the sentence "To achieve this nested provenances are downloaded recursively" intended like it is now? I think there's something missing...

Reply:
I have improved the sentence a bit to make clear why they are downloaded recursively.